pkgs/racket-doc/scribblings/guide/other-editors.scrbl

```diff
@@ -137,91 +137,69 @@ popular among Racketeers as well.
 
-@subsection[#:tag "vim-plugins"]{Plugins}
+@subsection[#:tag "vim-racket"]{Enhanced Racket Support}
 
-Alternatively, you can use a plugin such as @itemlist[
-    @item{@hyperlink["https://github.com/wlangstroth/vim-racket"]{wlangstroth/vim-racket}}
-    @item{@hyperlink["https://github.com/benknoble/vim-racket"]{benknoble/vim-racket}}
-]@margin-note{The major difference between the two is that the
-@tt{benknoble/vim-racket} fork supports more features out of the box and is
-updated more frequently.}
-to enable auto-detection, indentation, and syntax highlighting specifically for
-Racket files.
+Vim will detect your Racket files as Scheme out of the box. To get the
+additional features of the Racket filetype, consider installing the
+@tt{vim-racket} plugin from
+@hyperlink["https://github.com/benknoble/vim-racket"]{benknoble/vim-racket}. It
+enables auto-detection of Racket files on top of enhanced indentation and syntax
+highlighting. Vim's default support comes from a subset of this plugin;
+installing it yourself provides additional features.
 
-These plugins work by setting the @tt{filetype} option based on the @(hash-lang)
+The @tt{vim-racket} plugin detects the @tt{filetype} option based on the @(hash-lang)
 line. For example:@itemlist[
-    @item{A file starting with @code{#lang racket} or @code{#lang racket/base}
-    has @tt{filetype} equal to @tt{racket}.}
-    @item{A file starting with @code{#lang scribble/base} or @code{#lang scribble/manual}
-    has @tt{filetype} equal to @tt{scribble}.}
+    @item{A file starting with @code{#lang racket} or @code{#lang racket/base} has @tt{filetype} equal to @tt{racket}.}
+    @item{A file starting with @code{#lang scribble/base} or @code{#lang scribble/manual} has @tt{filetype} equal to @tt{scribble}.}
 ]
-Depending on which plugin you have, modifiers like @code{at-exp} may also be
-ignored, so that @code{#lang at-exp racket} is still a @tt{filetype} of
-@tt{racket}.
-
-This approach is more flexible but may lead to more work. Since each
-@(hash-lang) has its own @tt{filetype}, options, syntax highlighting, and other
-features need to be configured for each filetype. This can be done via the
-standard @tt{ftplugin} mechanism. See for example @tt{:help ftplugin-overrule}
-and @tt{:help ftplugin}: place your options for @tt{<lang>} in
-@filepath{~/.vim/after/ftplugin/<lang>.vim}
-(@filepath{$HOME/vimfiles/after/ftplugin/<lang>.vim} on MS-Windows). Similarly,
-syntax files follow the standard mechanism documented in @tt{:help syntax}.
-
-Both plugins come with configuration for Racket
-(and possibly other @(hash-lang)s) as @tt{ftplugin}s. To enable them, use the
-@tt{:filetype} command as documented in @tt{:help :filetype}. You likely want to
-turn on filetype plugins (@tt{:help :filetype-plugin-on}) and filetype indent
-plugins (@tt{:help :filetype-indent-on}).
+
+The @tt{vim-racket} plugin comes with configuration for Racket and some other
```

Comment:
nit: "The plugin includes configuration for Racket ..." since we've already mentioned the name of the plugin.